rs/embedders/src/wasmtime_embedder.rs

```diff
@@ -1,3 +1,8 @@
+// False positive clippy lint.
+// Issue: https://github.com/rust-lang/rust-clippy/issues/12856
+// Fixed in: https://github.com/rust-lang/rust-clippy/pull/12892
+#![allow(clippy::needless_borrows_for_generic_args)]
```

Comment:
Could we make a Jira item to remove this allow (and the two other cases under `rs/embedders`) on the next rustc version bump? I just don't want us to forget it.

Reply:
I have it noted down for the next upgrade, but created a JIRA ticket for it as well now. NET-1737